tests/accessibility/index.test.js

```diff
@@ -0,0 +1,361 @@
+  '/pl/course/:course_id/question/:question_id/submission/:submission_id/file/*',
+  '/pl/course/:course_id/question/:question_id/text/:filename',
+  '/pl/news_item/:news_item_id/*',
+
+  // Renders partial HTML documents, not a full page.
+  '/pl/course_instance/:course_instance_id/instance_question/:instance_question_id/variant/:variant_id/submission/:submission_id',
+  '/pl/course_instance/:course_instance_id/instructor/assessment/:assessment_id/manual_grading/instance_question/:instance_question_id/variant/:variant_id/submission/:submission_id',
+  '/pl/course_instance/:course_instance_id/instructor/instance_admin/assessments/stats/:assessment_id',
+  '/pl/course_instance/:course_instance_id/instructor/question/:question_id/preview/variant/:variant_id/submission/:submission_id',
+  '/pl/course_instance/:course_instance_id/assessment_instance/:assessment_instance_id/time_remaining',
+  '/pl/course/:course_id/question/:question_id/preview/variant/:variant_id/submission/:submission_id',
+
+  // These pages just redirect to other pages and thus don't have to be tested.
+  '/pl/course_instance/:course_instance_id/instructor/assessment/:assessment_id/manual_grading/assessment_question/:assessment_question_id/next_ungraded',
+  '/pl/course_instance/:course_instance_id/instructor/loadFromDisk',
+  '/pl/course_instance/:course_instance_id/loadFromDisk',
+  '/pl/course/:course_id/file_transfer/:file_transfer_id',
+  '/pl/course/:course_id/loadFromDisk',
+  '/pl/loadFromDisk',
+  '/pl/oauth2callback',
+  '/pl/oauth2login',
+
+  // TODO: enable once the following PR is merged:
+  // https://github.com/PrairieLearn/PrairieLearn/pull/7382
+  '/pl/course/:course_id/effectiveUser',
+
+  // TODO: add tests for file editing/viewing.
+  /\/file_edit\/\*$/,
+  /\/file_view\/\*$/,
+
+  // TODO: add tests for job sequence pages.
+  /\/:job_sequence_id$/,
+
+  // TODO: add tests for workspace pages. These will require us to open a question
+  // in order to create a workspace.
+  // TODO: open a question and create a workspace so we can test this page.
+  /^\/pl\/workspace\//,
+
+  // TODO: run a query so we can test this page.
+  '/pl/administrator/query/:query',
+
+  // TODO: create an assessment instance and create an instance question so we can test these pages.
+  '/pl/course_instance/:course_instance_id/assessment_instance/:assessment_instance_id',
+  '/pl/course_instance/:course_instance_id/instance_question/:instance_question_id',
+  '/pl/course_instance/:course_instance_id/instructor/assessment_instance/:assessment_instance_id',
+  '/pl/course_instance/:course_instance_id/instructor/assessment/:assessment_id/manual_grading/assessment_question/:assessment_question_id',
+  '/pl/course_instance/:course_instance_id/instructor/assessment/:assessment_id/manual_grading/instance_question/:instance_question_id',
+
+  // TODO: submit an answer to a question so we can test this page.
+  '/pl/course_instance/:course_instance_id/instructor/grading_job/:job_id',
```

Comment:
I'm currently inclined to leave these for the future, but will address them in this PR if folks feel strongly.

Reply:
I'm fine with leaving them until later